mne/viz/ica.py

```diff
@@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
     exclude_labels = list()
     for ii in picks:
         if ii in exclude:
-            line_label = 'ICA %03d' % (ii + 1)
```

Comment:
Sure. I've attached the ECG/EOG source plots with old code (appended by "_old.pdf") - and with "IC #%d#" % (ii) (appended by "_new.pdf").
And then I've also attached the topo-overview (where the ICA components are 0-indexed, plotted with plot_components). It should be evident from this that the EOG component is IC #009 and the ECG component is IC #013.
Is this sufficient?

[ica_source_ecg_new.pdf](https://github.com/mne-tools/mne-python/files/455569/ica_source_ecg_new.pdf)
[ica_source_ecg_old.pdf](https://github.com/mne-tools/mne-python/files/455570/ica_source_ecg_old.pdf)
[ica_source_eog_new.pdf](https://github.com/mne-tools/mne-python/files/455572/ica_source_eog_new.pdf)
[ica_source_eog_old.pdf](https://github.com/mne-tools/mne-python/files/455571/ica_source_eog_old.pdf)
[ica_topos_eog9_ecg13.pdf](https://github.com/mne-tools/mne-python/files/455573/ica_topos_eog9_ecg13.pdf)